gym_solo/core/rewards.py

```diff
@@ -39,8 +72,13 @@ class RewardFactory:
   temporal decay), then it's probably in your best interest to implement that
   in a custom Reward.
   """
-  def __init__(self):
-    """Create a new RewardFactory."""
+  def __init__(self, client: bullet_client.BulletClient):
+    """Create a new RewardFactory.
+    
+    Args:
+      client (bullet_client.BulletClient): Sandboxed Pybullet client.
+    """
+    self._client = client
     self._rewards: List[_WeightedReward] = []
```

Comment:
What is this syntax for?

Reply:
Adding a type comment on the rewards list?

Reply:
I am just curious about why you use 'List[_WeightedReward]' instead 'List'. Does it specify that it is a list of _weightedReward objects?

Reply:
Ahh gotcha. Yep that's exactly what it does--makes using vscode a lot easier too cause hints get a lot smarter xD

Reply:
Hahha nice